terraform/aks/workspace_variables/qa_aks.tfvars.json

```diff
@@ -0,0 +1,30 @@
+{
+  "env_config": "qa_aks",
+  "app_environment": "qa",
+  "key_vault_name": "s189t01-ptt-qa-kv",
+  "key_vault_app_secret_name": "PUBLISH-APP-SECRETS-QA",
+  "key_vault_infra_secret_name": "BAT-INFRA-SECRETS-QA",
+  "cluster": "test",
+  "namespace": "bat-qa",
+  "db_sslmode": "prefer",
+  "resource_group_name": "s189t01-ptt-qa-rg",
+  "main_app": {
+    "main": {
+      "replicas": 2
+    }
+  },
+  "worker_apps": {
+    "worker": {
+      "replicas": 1
+    }
+  },
+  "enable_find": true,
+  "additional_hostnames": [
+    "qa2.publish-teacher-training-courses.service.gov.uk",
+    "qa2.find-postgraduate-teacher-training.service.gov.uk",
+    "qa2-assets.find-postgraduate-teacher-training.service.gov.uk",
+    "qa2.api.publish-teacher-training-courses.service.gov.uk",
+    "find-qa.test.teacherservices.cloud"
+  ],
+  "enable_monitoring": false
+}
```

Comment:
azure_patch_schedule for redis (although can go in the .tf file

Reply:
Added to tf file